fairlearn/postprocessing/_roc_curve_utilities.py

```diff
@@ -8,11 +8,21 @@
 from ._threshold_operation import ThresholdOperation
 
 
-def _get_roc(data, x_grid, attribute, flip=True):
+def _get_roc(data, sensitive_feature_value, flip=True):
```

Comment:
This looks like more than a doc change - you're going from two arguments to one.

Reply:
x_grid wasn't used as it turns out!